lib/model/Fixture.mjs

```diff
@@ -534,4 +543,20 @@ export default class Fixture {
 
     return this._cache.modes;
   }
+
+  /**
+   * Helper function to merge array properties of an array of items.
+   * @param {!Array.<object>} items The items whose property values should be merged.
+   * @param {!string} property The items' property name to merge.
+   * @returns {!Array} Array of concatenated property values.
+   */
+  _concatArrayProperty(items, property) {
```

Comment:
Better move this outside the class as it does not need a reference to `this`. Then it automatically is "really" private.